x-pack/plugins/lens/public/shared_components/visual_options_popover/visual_options_popover.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiSpacer } from '@elastic/eui';
+import { ToolbarPopover } from '../toolbar_popover';
+import { MissingValuesOptions } from './missing_values_option';
+import { LineCurveOption } from './line_curve_option';
+import { XYState } from '../..';
+
+export interface VisualOptionsPopoverProps {
+  state: XYState;
+  setState: (newState: XYState) => void;
+  isCurveTypeEnabled: boolean;
+  isValueLabelsEnabled: boolean;
+  isFittingEnabled: boolean;
+}
+
+export const VisualOptionsPopover: React.FC<VisualOptionsPopoverProps> = ({
+  state,
+  setState,
+  isCurveTypeEnabled,
+  isValueLabelsEnabled,
+  isFittingEnabled,
+}) => {
+  return (
+    <ToolbarPopover
+      title={i18n.translate('xpack.lens.shared.curveLabel', {
+        defaultMessage: 'Visual options',
```

Comment:
Any thoughts on changing `Visual options` to `Appearance options`?

Reply:
Honestly to me seems like Visual and Appearance both are loaded words, but i am not a native speaker :D 
It could be just simply `Style options`? then again style could be more of a css thing. i am 50/50 on Visual and Appearance. may be slightly in favor of Visual.

Reply:
No opinion on this, previously Gail would do a pass over all newly introduced wording before the release. I think we can go either way on this PR and let her check before we push it out.